supervision/annotators/core.py

```diff
@@ -315,6 +316,86 @@ def annotate(self, scene: np.ndarray, detections: Detections) -> np.ndarray:
         return scene
 
 
+class CircleAnnotator(BaseAnnotator):
+    """
+    A class for drawing circle on an image using provided detections.
+    """
+
+    def __init__(
+        self,
+        color: Union[Color, ColorPalette] = ColorPalette.default(),
+        thickness: int = 4,
+        color_map: str = "class",
+    ):
+        """
+        Args:
+            color (Union[Color, ColorPalette]): The color or color palette to use for
+                annotating detections.
+            thickness (int): Thickness of the circle line.
+            color_map (str): Strategy for mapping colors to annotations.
+                Options are `index`, `class`, or `track`.
+        """
+
+        self.color: Union[Color, ColorPalette] = color
+        self.thickness: int = thickness
+        self.color_map: ColorMap = ColorMap(color_map)
+
+    def annotate(
+        self,
+        scene: np.ndarray,
+        detections: Detections,
+    ) -> np.ndarray:
+        """
+        Draws circle on the frame using the detections provided.
+        Args:
+            scene (np.ndarray): The image on which the circle will be drawn
+            detections (Detections): The detections for which the
+                circle will be drawn
+        Returns:
+            np.ndarray: The image with the circle drawn on it
+        Example:
+            ```python
+            >>> import supervision as sv
+
+            >>> image = ...
+            >>> detections = sv.Detections(...)
+
+            >>> circle_annotator = sv.CircleAnnotator()
+            >>> annotated_frame = circle_annotator.annotate(
+            ...     scene=image.copy(),
+            ...     detections=detections
+            ... )
+            ```
```

Comment:
Could you add this link to an example annotated frame in docs? `https://media.roboflow.com/supervision-annotator-examples/circle-annotator-example.png` In a similar way, we have done it for other annotators.

Reply:
@SkalskiP  I was going to ask that, awesome thank you !!

Reply:
No worries ;) let me know once you update it. ;)